suitcase/mongo_normalized/__init__.py

```diff
@@ -90,6 +92,23 @@ def __call__(self, name, doc):
         sanitized_doc = event_model.sanitize_doc(doc)
         return super().__call__(name, sanitized_doc)
 
+    def update(self, name, doc):
+        if name in ['run_start', 'run_stop']:
```

Comment:
We refer to these as "Run Start" and "Run Stop" documents, but in the code they are always marked as `'start'` and `'stop'`.